define/jsonhelper.go

```diff
@@ -124,14 +134,20 @@ type DataCreatePoll struct {
 	// name of LAO
 	Name string
 	//Creation Date/Time
-	Creation      int64  //  Unix timestamp (uint64)
-	Last_modified int64  //timestamp
-	Location      string //optional
+	Creation int64 //  Unix timestamp (uint64)
+	//Last_modified int64  //timestamp
+	Location string //optional
 	//Organiser: Public Key
 	Start int64  /* Timestamp */
 	End   int64  /* Timestamp, optional */
 	Extra string /* arbitrary object, optional */
 }
+type DataWitnessMessage struct {
+	Object     string
+	Action     string
+	Message_id string
+	Signature  string
+}
 
```

Comment:
below is a lot of logic that goes way beyond what the package name refers to `define`.
You might consider splitting this package into different packages, one dealing with just the JSON-RPC layer, another dealing with messaging, and the final one dealing with the actual message types. In that case, the definitions would be in each package.

Reply:
so we splitted the package define in 5 packages :

* security
* events (old dataDefinitions.go)
* lib (general functions like Decode and Find)
* parser (+composer to read / write messages)
* message struct representing JSON sent/received messages

We'd be glad if you could go and take a look at it :)

Reply:
Happy to, where is that?